docs/contracts.rst

```diff
@@ -834,6 +845,9 @@ Methods
     `More information about archival nodes here <https://ethereum.stackexchange.com/a/84200/620>`_.
 
 .. py:method:: ContractFunction.estimateGas(transaction, block_identifier=None)
+    .. warning:: Deprecated: This method is deprecated in favor of :class:`~estimate_gas`
```

Comment:
I think this needs one more space in between 847 and 848. It's showing up like this: 
<img width="730" alt="image" src="https://user-images.githubusercontent.com/6540608/168338734-0e62d6aa-50b3-4378-98b5-1c844ab16f93.png">

Reply:
fixed